x-pack/plugins/infra/public/alerting/inventory/components/alert_flyout.tsx

```diff
@@ -39,7 +38,7 @@ export const AlertFlyout = ({ options, nodeType, filter, visible, setVisible }:
           options,
           nodeType,
           filter,
-          customMetrics,
+          customMetrics: inventoryPrefill?.customMetrics ?? [],
```

Comment:
This is needed for the story (otherwise the storybook shows an error). Not sure if we can do something better there to avoid this change.

Reply:
The problem is that the `inventoryPrefill` is `undefined` when the component is used in the storybook context- that's why I used `inventoryPrefill?.customMetrics`. Your idea is good I will try it out